src/modules/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -1,13 +1,60 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 
-import { StacheNav, StacheNavLink } from '../nav';
+import { StacheNav, StacheNavLink, StacheNavService } from '../nav';
 
 @Component({
   selector: 'stache-breadcrumbs',
   templateUrl: './breadcrumbs.component.html',
   styleUrls: ['./breadcrumbs.component.scss']
 })
-export class StacheBreadcrumbsComponent implements StacheNav {
+export class StacheBreadcrumbsComponent implements StacheNav, OnInit {
   @Input()
   public routes: StacheNavLink[];
+
+  public constructor(
+    private navService: StacheNavService) { }
+
+  public ngOnInit(): void {
+    if (!this.routes) {
+      const activeRoutes = this.navService.getActiveRoutes();
+      this.routes = this.filterRoutes(activeRoutes);
+    }
+  }
+
+  private filterRoutes(activeRoutes: StacheNavLink[]): StacheNavLink[] {
+    const activeUrl = this.navService.getActiveUrl();
+    const root = activeRoutes[0];
+    let breadcrumbRoutes: StacheNavLink[] = [];
+
+    breadcrumbRoutes.push({
+      name: 'Home',
+      path: '/'
+    });
+
+    if (root.path !== '/') {
+      breadcrumbRoutes.push({
+        name: root.name,
+        path: root.path
+      });
+    }
+
+    const findActiveRoute = (routes: any[]) => {
```

Comment:
Is there a reason this method has to be defined as a const and live inside the filterRoutes method?  Would it not be better to have it live separate?